sql/catalyst/src/main/scala/org/apache/spark/sql/OutputMode.java

```diff
@@ -15,9 +15,10 @@
  * limitations under the License.
  */
 
-package org.apache.spark.sql.catalyst.analysis
+package org.apache.spark.sql;
```

Comment:
nit: move this file to sql/catalyst/src/main/**java**/org/apache/spark/sql/OutputMode.java